### PR TITLE
Adds a note to the first instruction to log into fuse online, specify…

### PR DESCRIPTION
**Pull request by briandooley**
…ing that it is the personal Fuse Online instance and not the shared instance

### CODE REVIEW

**Comment by briandooley** [comment]
@finp for review.

File: walkthroughs/1A-integrate-event-and-api-driven-apps/walkthrough.adoc [hunk 1]
```diff
@@ -127,6 +127,8 @@ It can be extended with custom connectors, using either OpenAPI definitions (Swa
 To create a connector for the RESTful API, register the OpenAPI definition as a *Customization*.
 
 . Log in to the link:{fuse-url}[Red Hat Fuse Online, window="_blank", id="{context}-1"] console.
++
+NOTE: This is your personal Fuse Online instance and not the shared instance available from link:/[home page].
```
Comment (pwright) [inline walkthroughs/1A-integrate-event-and-api-driven-apps/walkthrough.adoc]: maybe we need to finesse this, 'personal fuse' is not an agreed term to present to users (altho used alot in eng).
Options are:
- extend the vocab (get approval for 'personal fuse')
- change wording to match 'Getting Started', eg

NOTE: Starting this {walkthrough-name} provisioned an instance of Fuse Online which is not shared with other cluster users. You are not logging into the shared Fuse Online instance, available from the link:/[Applications menu] with this step.

**Review by pwright — CHANGES_REQUESTED** [review]

File: walkthroughs/1A-integrate-event-and-api-driven-apps/walkthrough.adoc (hunk 1 above)
Comment (briandooley) [reply walkthroughs/1A-integrate-event-and-api-driven-apps/walkthrough.adoc]: How about "exclusive", "particular", "unique" or a combination of these?

Comment (briandooley) [reply walkthroughs/1A-integrate-event-and-api-driven-apps/walkthrough.adoc]: Replacement text agreed.

> NOTE: You are not logging into the shared Fuse Online instance, available from the link:/[Applications menu] with this step.  Starting this {walkthrough-name} provisioned an instance of Fuse Online which is not shared with other cluster users.

**Review by pwright — APPROVED** [review]

**Pull request merged**